src/orquestra/sdk/_ray/_ray_logs.py

```diff
@@ -4,15 +4,19 @@
 """
 Class to get logs from Ray for particular Workflow, both historical and live.
 """
+import ast
 import glob
 import os
+import re
 import time
 import typing as t
 from dataclasses import dataclass
 from pathlib import Path
 
 from . import _client
 
+LINE_FORMAT: str = "{log.timestamp}\t{log.level} {log.filename} -- {log.message.logs}\t[{log.message.id}]"  # noqa E501
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick (non-blocking):** ‏We can look into this later, but this format looks a little odd to me.

```
2023-01-30 11:03:13,150 INFO task_executor.py:78 -- Task status [RUNNING]       [wf.wf.cd86d3f@invocation-0-task-a]
2023-01-30 11:03:13,157 INFO _log_adapter.py:108 -- hello       [wf.wf.cd86d3f@invocation-0-task-a]
2023-01-30 11:03:13,157 CRITICAL tasks.py:7 -- HELLO    [wf.wf.cd86d3f@invocation-0-task-a]
2023-01-30 11:03:13,169 INFO task_executor.py:78 -- Task status [RUNNING]       [wf.wf.cd86d3f@orquestra.sdk._ray._dag._make_ray_dag_node.locals._ray_remote]
2023-01-30 11:03:13,180 INFO task_executor.py:78 -- Task status [RUNNING]       [wf.wf.cd86d3f@orquestra.sdk._ray._dag._make_ray_dag.locals.handle_data_aggregation_error]
2023-01-30 11:03:10,935 INFO workflow_executor.py:86 -- Workflow job [id=wf.wf.cd86d3f] started.
2023-01-30 11:03:13,159 INFO workflow_executor.py:284 -- Task status [SUCCESSFUL]       [wf.wf.cd86d3f@invocation-0-task-a]
2023-01-30 11:03:13,171 INFO workflow_executor.py:284 -- Task status [SUCCESSFUL]       [wf.wf.cd86d3f@orquestra.sdk._ray._dag._make_ray_dag_node.locals._ray_remote]
2023-01-30 11:03:13,183 INFO workflow_executor.py:284 -- Task status [SUCCESSFUL]       [wf.wf.cd86d3f@orquestra.sdk._ray._dag._make_ray_dag.locals.handle_data_aggregation_error]
2023-01-30 11:03:13,187 INFO workflow_executor.py:122 -- Workflow 'wf.wf.cd86d3f' completes successfully.
2023-01-30 11:03:13,150 INFO task_executor.py:78 -- Task status [RUNNING]       [wf.wf.cd86d3f@invocation-0-task-a]
2023-01-30 11:03:13,157 INFO _log_adapter.py:108 -- hello       [wf.wf.cd86d3f@invocation-0-task-a]
2023-01-30 11:03:13,157 CRITICAL tasks.py:7 -- HELLO    [wf.wf.cd86d3f@invocation-0-task-a]
2023-01-30 11:03:13,169 INFO task_executor.py:78 -- Task status [RUNNING]       [wf.wf.cd86d3f@orquestra.sdk._ray._dag._make_ray_dag_node.locals._ray_remote]
2023-01-30 11:03:13,180 INFO task_executor.py:78 -- Task status [RUNNING]       [wf.wf.cd86d3f@orquestra.sdk._ray._dag._make_ray_dag.locals.handle_data_aggregation_error]
2023-01-30 11:03:10,935 INFO workflow_executor.py:86 -- Workflow job [id=wf.wf.cd86d3f] started.
2023-01-30 11:03:13,159 INFO workflow_executor.py:284 -- Task status [SUCCESSFUL]       [wf.wf.cd86d3f@invocation-0-task-a]
2023-01-30 11:03:13,171 INFO workflow_executor.py:284 -- Task status [SUCCESSFUL]       [wf.wf.cd86d3f@orquestra.sdk._ray._dag._make_ray_dag_node.locals._ray_remote]
2023-01-30 11:03:13,183 INFO workflow_executor.py:284 -- Task status [SUCCESSFUL]       [wf.wf.cd86d3f@orquestra.sdk._ray._dag._make_ray_dag.locals.handle_data_aggregation_error]
```